src/renderer/components/FileDisplay/FileDisplay.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useMemo } from 'react';
+import FileIcon from '@mui/icons-material/InsertDriveFileOutlined';
+import FolderIcon from '@mui/icons-material/FolderOpenTwoTone';
+import { Typography, useTheme } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { DirectoryEntryInfo } from '../../../shared/types/api';
+import * as Styled from './style';
+import { widths } from './Columns';
+
+interface Props {
+  entry: Required<DirectoryEntryInfo>;
+  treeLevel?: number;
+}
+
+const [statuses, colors] = [
+  ['added', 'modified', 'deleted', 'moved', 'unchanged'],
+  ['green', 'blue', 'red', 'orange', 'primary'],
+] as const; // placeholder
+
+const FileDisplay = ({ entry, treeLevel }: Props) => {
+  const { t } = useTranslation();
+  const dateM = new Date(entry.details.dateModifiedMs);
+  const statusIndex = useMemo(() => Math.floor(Math.random() * 5), []);
```

Comment:
Sometimes the placeholder status is not shown in the cell but I suppose that's because of this randomization going out of bound of the array maybe?

Reply:
There was no logic bug, but empty strings in translations - I could not find a good word for 'unchanged', so did an experiment with just leaving them blank and forgot to do something with it (e.g. a conditional in the component instead of an empty translation) before doing the PR. Filled them in for now.